qemu/tests/tpm_check_buffer_size.py

```diff
@@ -0,0 +1,54 @@
+import os
+from shutil import rmtree
+
+from avocado.utils import git
+from avocado.utils import process
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run Check size of buffer before accessing it in libtpms
+    upstream test tree
+
+    1. Download and install libtpms-devel rpm
+    2. Clone libtpms test tree
+    3. Comple and execute test cases
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    s, o = process.getstatusoutput("rpm -q libtpms-devel", shell=True)
+
+    if s:
+        try:
+            libtpms_rpm = process.system_output("rpm -q libtpms", shell=True)
+            libtpms_devel_rpm = (
+                libtpms_rpm.split(b'-', 1)[0] +
+                b'-devel-' +
+                libtpms_rpm.split(b'-', 1)[1]).decode()
+            process.system(
+                "brew download-build --rpm %s" %
+                libtpms_devel_rpm, shell=True)
+            libtpms_devel_rpm = libtpms_devel_rpm + '.rpm'
+            process.system("rpm -i %s" % libtpms_devel_rpm, shell=True)
```

Comment:
Better to remove the rpm in the finish.

Reply:
Add remove step in the finally section .